rs/tests/driver/src/driver/farm.rs

```diff
@@ -94,7 +94,7 @@ impl Farm {
         &self,
         group_base_name: &str,
         group_name: &str,
-        ttl: Duration,
+        ttl: Option<Duration>,
```

Comment:
I'm wondering if this implementation is a little risky. What if a user accidentally doesn't specify the duration and the testnet becomes eternal? Would it be better to have an explicit input for a testnet to be eternal?

Reply:
This whole feature is indeed a bit risky because it can lead to resource exhaustion. So what is important is that we monitor eternal testnets. We could consider adding a dedicated graph to the [Farm Dashboard](https://grafana.ch1-obsstage1.dfinity.network/d/uwEFG_yGk/testing-dashboard?orgId=1&from=now-3h&to=now&refresh=10s) that tracks eternal testnets.

With regards to this function signature, it should model the same signature as [the corresponding Farm endpoint](https://github.com/dfinity-lab/farm/blob/master/backend/Farm/WebServer/API.hs#L206-L209). Do note that the only way to enable this feature is to pass `--test_arg=--no-group-ttl"` to the bazel invocation (or `--no-ttl` to the `ict testnet create` invocation. So it's hard to accidentally do that.